bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -154,11 +154,18 @@ public BookKeeperAdmin(String zkServers) throws IOException, InterruptedExceptio
      *             Throws this exception if there is an error instantiating the
      *             BookKeeper client.
      */
+    public BookKeeperAdmin(ClientConfiguration conf, ServerConfiguration serverConf) throws IOException, InterruptedException, BKException {
+        // Create the BookKeeper client instance
+        bkc = new BookKeeper(conf);
+        ownsBK = true;
+        this.lfr = new LedgerFragmentReplicator(bkc, NullStatsLogger.INSTANCE, serverConf);
```

Comment:
probably it is possible to build a ServerConfiguration from a ClientConfiguration

Reply:
I saw BookkeeperAdmin would be used by external, like pulsar. I don't know If I change the constructor, it may have affect